skyreader/plot/plotting_tools.py

```diff
@@ -16,15 +16,17 @@
 
 matplotlib.use('agg')
 
-def format_fits_header(event_id_tuple, mjd, ra, dec, uncertainty, llh_map):
+def format_fits_header(
+    event_id_tuple, mjd, ra, dec, uncertainty, llh_map,
+):
     """Prepare some of the relevant event information for a fits file
     header."""
     run_id, event_id, event_type = event_id_tuple
 
     if llh_map:
         uncertainty_comment = 'Change in 2LLH based on Wilks theorem'
     else:
-        uncertainty_comment = 'Probability-per-pixel (all pixels with same area)'
+        uncertainty_comment = 'Area with 50%(90%) total probability'
```

Comment:
Is there a typo here, `50%(90%)`?

Reply:
Later on this uncertainty comment is usedd here:
```
        ('COMMENTS', '50%(90%) uncertainty location' \
            + ' => ' + uncertainty_comment),
```
I think the new version of the comment is more appropriate.